eclipse-collections/src/main/java/org/eclipse/collections/impl/factory/Multimaps.java

```diff
@@ -291,6 +294,65 @@ private ImmutableBagMultimapFactory()
                 return hashBagMultimap.toImmutable();
             }
         }
+
+        public static final class ImmutableSortedBagMultimapFactory
+        {
+
+           private ImmutableSortedBagMultimapFactory()
+           {
+
+           }
+
+           public <K,V>ImmutableSortedBagMultimap<K,V> of(Comparator<V> comparator)
```

Comment:
I think the generics should allow type variance. So `Comparator<? super V>` here and in a few other places. There are also minor whitespace problems that are going to break our CheckStyle build. Should be obvious from the travis builds I hope.